### PR TITLE
New Rule: Credential Phishing: Fake Password Expiration

### PR DESCRIPTION
**Pull request by morriscode**

### CODE REVIEW

File: credential_phishing_fake_password_expiration.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+name: "Credential Phishing: Fake Password Expiration from New and Unsolicited sender"
+description: "This rule looks for password expiration verbiage in the subject and body. It requires between 1 and 9 links in the body. The rule uses NLU in addition to statically specified term anchors. High trust senders are also negated. "
```
Comment (bfilar) [inline credential_phishing_fake_password_expiration.yml:2]: Extra space in `description`

**Review by bfilar — APPROVED** [review]

**Comment by morriscode** [comment]
Seeing some FP's on legitimate services. Typically these are falling under "outliers" as they send very infrequently and seem to be spawned by a user request to reset a password.

**Comment by morriscode** [comment]
This rule will not enter testing.

**Pull request closed without merging**